rise/static/main.js

```diff
@@ -248,6 +248,39 @@ function disconnectOutputObserver() {
   }
 }
 
+function addHeaderFooterOverlay(config) {
+    let overlay = config.get_sync('overlay');
+    let header =  config.get_sync('header');
+    let footer =  config.get_sync('footer');
+    let backimage =  config.get_sync('backimage');
```

Comment:
I think we need to put default values here https://github.com/parmentelat/RISE/blob/master/rise/static/main.js#L25, for these options. Don't you think?

Reply:
well, I'm not quite sure now, but I think at the time I considered it but rules it out because of the config logic; IIRC having overlay defined kind of hides the ability to use header/footer/backimge (or it it the other way around?)

So probably having them in a comment could be helpful, but more than that I don't know

Reply:
OK, a comment sound like a good compromise.